src/Dex.tsx

```diff
@@ -40,8 +50,13 @@ function makePKMN(p: Pokemon, i: number) {
   );
   const displayNumber = "#" + String(p.number).padStart(3, "0");
   const style = { minHeight: "100px" };
+  const onClick = () => {
+    gChangeTab(1);
+    gupdateType1(p.types[0]);
+    gupdateType2(p.types[1]);
```

Comment:
This right now is crashing if the Pokemon doesn't have a second type, you should use "none" as a fallback value.